tests/suites/test_suite_rsa.function

```diff
@@ -45,7 +46,13 @@ void mbedtls_rsa_pkcs1_sign( data_t * message_str, int padding_mode,
 
     TEST_ASSERT( mbedtls_rsa_import( &ctx, &N, &P, &Q, NULL, &E ) == 0 );
     TEST_ASSERT( mbedtls_rsa_get_len( &ctx ) == (size_t) ( mod / 8 ) );
-    TEST_ASSERT( mbedtls_rsa_complete( &ctx ) == 0 );
+
+    res = mbedtls_rsa_complete( &ctx );
+#if defined(MBEDTLS_RSA_ALT)
+    TEST_ASSUME( res != MBEDTLS_ERR_RSA_UNSUPPORTED_OPERATION );
```

Comment:
This function has no business failing on an alternative implementation. Instead, we should provide all the parameters (D, DP, DQ, U). If there's some urgency in making the test pass on some alternative implementations, I won't block this PR, but I don't like it because it increases the test gap. We should fix it ASAP. Tracked in #1985

Reply:
Gilles, I don't follow. We have tests which provide all parameters, and we have tests which provide them only partially, to test the functionality of automatic parameter derivation if it's supported. If not, these tests shall be skipped, which is what this PR does.

Can you please elaborate, and also explain the reason why you changed your mind since your last approval?

Reply:
Do we have tests with all the parameters? Oh, I guess we do, in `test_suite_pkcs1_v15` and `test_suite_pkcs1_v21`. Can you please add a comment with a reference to these other files?

As I said, I won't block the PR for that. The only change that I'm requesting is to use an enum for pass/fail/skip instead of an int.